app/src/pages/outside/registrationPage/registrationPage.jsx

```diff
@@ -111,6 +112,12 @@ const TokenErrorSection = ({ tokenProvided }) => (
       <a className={cx('backlink')} href={referenceDictionary.rpLanding}>
         ReportPortal.io
       </a>
+      <br />
+      <FormattedMessage id={'RegistrationPage.or'} defaultMessage={'or '} />
```

Comment:
You set default message as <text>+space so you have `or[space]`, but in translation you have only <text> without any spaces(after here and before in 120line). So please recheck how it looking after selecting different lang(not eng)